ChangeFeedLabSolution/ChangeFeedProcessor.cs

```diff
@@ -35,7 +36,7 @@ public static class ChangeFeedProcessor
         /// <param name="documents"> Modified records from Cosmos DB collections. </param>
         /// <param name="log"> Outputs modified records to Event Hub. </param>
         [FunctionName("ChangeFeedProcessor")]
-        public static void Run(
+        public static async void Run(

```

Comment:
This should return `Task`.    In general, no method should be returning `void` if marked `async` outside of a very few special corner cases.   Here, Functions supports the correct signature.   _([ref](https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-class-library?tabs=v2%2Ccmd#async))_